### PR TITLE
[minor] Support switch MAS legacy install to OLM

### PR DESCRIPTION
**Pull request by andrercm**
Implementation for `Integrate sre_devops.suite_switch_to_olm into mas_devops`: https://github.ibm.com/wiotp/tracker/issues/10624

### CODE REVIEW

**Review by durera — CHANGES_REQUESTED** [review]
This also needs to be able to transition all installed MAS apps to OLM, not just the ibm-mas operator in mas-xxx-core.

File: ibm/mas_devops/roles/suite_switch_to_olm/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+suite_switch_to_olm
+=============
+This role provides capabilities to switch a MAS instance from legacy deployment using mas-install.sh installer to OLM/Channel subscription install.
+
+Role Variables
+--------------
+- `mas_catalog_source` Defines the catalog to be used to install MAS channel subscription. Default to be used is `ibm-operator-catalog`.
```
Comment (durera) [inline ibm/mas_devops/roles/suite_switch_to_olm/README.md:7]: Can we convert these to the new (well not really new anymore) format for the the variable list

```
### var name
Description paragraph

- Required/Optional
- Environment Variable: `xxx`
- Default: `xxx`
```

**Review by durera — CHANGES_REQUESTED** [review]
Changelog update missing too :+1:

File: ibm/mas_devops/roles/suite_switch_to_olm/README.md (hunk 1 above)
Comment (andrercm) [reply ibm/mas_devops/roles/suite_switch_to_olm/README.md]: Fixed it

**Pull request merged**